z02-parts-of-a-graph.md

```diff
@@ -224,19 +226,18 @@ We'll come across more scales later.
 
 In our example, up top, we have these nice labels and tick marks. This is
 something D3 can do for us. We can build an axis, and apply it to a scale. We
-say, "hey, I want to build an axis that".
+say, "hey, I want to build an axis that labels the plot".
```

Comment:
derp, good catch. I think I meant "an axis that describes the range of data that I have"